src/application.php

```diff
@@ -45,6 +49,13 @@
         }
 
         $configuration = readConfiguration($inputFile);
+
+        if (!$input->getOption('ignore-override') && file_exists($overrideFile)) {
+            $override = readConfiguration($overrideFile);
+
+            $configuration = array_merge_recursive($configuration, $override);
```

Comment:
@astrospaj I implemented a merge strategy that seems to work pretty well. Everything is merge except form volumes and links where I coded a conflict resolution.

If two volume map to the same container directory, only the last one will be present. The same applies for link aliases.